minio_exporter.go

```diff
@@ -171,6 +172,197 @@ func collectServerStats(e *MinioExporter, ch chan<- prometheus.Metric) {
 				nil),
 			prometheus.GaugeValue,
 			float64(connStats.TotalOutputBytes), host)
+		httpStats := stats.Data.HTTPStats
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, "http", "total_count_heads"),
+				"Minio total input bytes received",
+				[]string{"minio_host"},
+				nil),
+			prometheus.GaugeValue,
+			float64(httpStats.TotalHEADStats.Count), host)
+
+		f, _ := strconv.ParseFloat(httpStats.TotalHEADStats.AvgDuration, 64)
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, "http", "total_avg_duration_heads"),
+				"Minio total input bytes received",
+				[]string{"minio_host"},
+				nil),
+			prometheus.GaugeValue,
+			float64(f), host)
+
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, "http", "success_count_heads"),
+				"Minio total output bytes received",
+				[]string{"minio_host"},
+				nil),
+			prometheus.GaugeValue,
+			float64(httpStats.SuccessHEADStats.Count), host)
+
+	    g, _ := strconv.ParseFloat(httpStats.SuccessHEADStats.AvgDuration, 64)
```

Comment:
I'm not sure how to use this metric which is returned as a string https://github.com/minio/minio/blob/83abad0b3732da4bff851ef9c8a1bbabc9802ca2/cmd/http-stats.go#L100